topics/dart/index.md

```diff
@@ -0,0 +1,14 @@
+---
+display_name: Dart
+topic: dart
+github_url: https://github.com/dart
+logo: dart.png
+related: flutter
+released: Oct 2011
+short_description: Flutter is an open-source mobile application development SDK created by Google.
```

Comment:
The `short_secription` you added is about with flutter and should be about Dart.